apps/vs-code-designer/src/CHANGELOG.md

```diff
@@ -0,0 +1,187 @@
+# Change Log
+
+## [1.0.45] - 2022-12-13
+
+### Changed
+
+- Fixed Bug where parameterized connections were not being sent when deployed.
+- Fixed Bug about not being able to debug nuget based app when .NET 6 version is installed.
+- Fixed Bug where connections interpolations were not working when using parameters.json.
+- Fixed Bug when deleted logic app site parameter is null.
+
+## [1.0.44] - 2022-11-09
+
+### Changed
+
+- Added support to Functions Core Tools version 4 and .NET version 6.
+- Added support for dynamic calls in local projects and cloud apps.
+- Added support Open Designer and Overview commands via command palette.
+- Added telemetry in Monitoring View.
```

Comment:
@ecfan Monitoring View is the view user gets when it clicks in any run of overview. Like this one:
![image](https://user-images.githubusercontent.com/102700317/213242814-5b0e7fe4-d94a-40a1-a1dc-bde9269db1a7.png)

Reply:
Ah, the docs refer to this experience as "review workflow run history": 

https://learn.microsoft.com/azure/logic-apps/monitor-logic-apps?tabs=standard#review-workflow-run-history

Can we rephrase as the following text, no caps because there's no UX name or title. Although, the doc uses the word "monitor", the customer isn't really monitoring in real-time, but after the fact?

`- Added telemetry to workflow run history view.`